cli/src/test/java/com/devonfw/tools/ide/commandlet/HelpCommandletTest.java

```diff
@@ -0,0 +1,93 @@
+package com.devonfw.tools.ide.commandlet;
+
+import com.devonfw.tools.ide.context.AbstractIdeContextTest;
+import com.devonfw.tools.ide.context.IdeContext;
+import com.devonfw.tools.ide.log.IdeLogLevel;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Integration test of {@link HelpCommandlet}.
+ */
+public class HelpCommandletTest extends AbstractIdeContextTest {
+
+  /**
+   * Test of {@link HelpCommandlet} does not require home.
+   */
+  @Test
+  public void testThatHomeIsNotReqired() {
+
+    // arrange
+    String path = "workspaces/foo-test/my-git-repo";
+    IdeContext context = newContext("basic", path, false);
```

Comment:
for such simple tests, you can also use `IdeTestContextMock.get()`

Reply:
Done